apptools/preferences/tests/test_preferences_helper.py

```diff
@@ -53,11 +55,17 @@ def setUp(self):
         # The filename of the example preferences file.
         self.example = resource_filename(PKG, 'example.ini')
 
+        # A temporary directory that can safely be written to.
+        self.tmpdir = tempfile.mkdtemp()
+
         return
 
     def tearDown(self):
         """ Called immediately after each test method has been called. """
 
+        # Remove the temporary directory.
+        os.removedirs(self.tmpdir)
```

Comment:
See #178; this should be changed to `rmdir` (though the solution I generally prefer is to use `shutil.rmtree`; that way you don't have to remove anything that was created inside the temporary directory first).